### PR TITLE
Corrections, improvements in GUI and prompt

### PR DESCRIPTION
**Pull request by ghost**
Changes made:
Now the boxes that hold the examples have a shadow to make them look cooler.
Tsx syntax highligthing configured.
Inputs are blocked when you have sent the request and are expecting a response.
Delete all button now removes every text on the screen.

IN PROGRESS:
Current prompt its still beeing improved. Some unstable behavior from GPT has been discovered with the information returned.

### CODE REVIEW

**Comment by ghost** [comment]
UPDATE:
The source of issues of the unstable responses from GPT seems to be the prompt. The prompt has been modified, i erased some text from it to make it less precise and let GPT have more control over which code to return, for now this has worked. Still testing to catch more errors.

**Review by oscarfgq — APPROVED** [review]

**Comment by ghost** [comment]
UPDATE: 
This new prompt seems to work fine, I tested it with diferent code samples and it returned the expected response.
Converting draft to PR.

File: src/app/components/Examples.tsx [hunk 1]
```diff
@@ -23,14 +27,15 @@ export function Examples({ send }: ExampleTypes){
             </div>
             <div id="examples-btns" className="flex flex-row justify-start flex-initial basis-1/12">
                 <button type="button"
```
Comment (geoperez) [inline src/app/components/Examples.tsx]: Extract the botton to a new componet.

File: src/store/inputStore.ts [hunk 2]
```diff
@@ -30,14 +30,16 @@ interface ArrInterface{
...
-//store for array that stores user's input and results.
+//array that stores user's inputs and results.
 //these are the examples that are fed to GPT. 
 export const userInputsArr = create<ArrInterface>((set)=>({
     examples: [],
     updateUserInputArr: (input:ShotsTypes)=> set((status:ArrInterface)=>({
         examples:[...status.examples, {input: input.input, result: input.result}]
     })),
     deleteUserInputArr:() => set((status:any)=>{
+        userInputStore.getState().updateUserInput('');
```
Comment (geoperez) [inline src/store/inputStore.ts]: If you are affecting another store, why don't you merge them into a single one?

**Review by geoperez — CHANGES_REQUESTED** [review]

**Review by AntonioGGovea — APPROVED** [review]

**Review by geoperez — APPROVED** [review]

**Pull request merged**